news/2024/2024-05-07-osutaiko-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,333 @@
+> 
+> You can find me in some known rhythm games such as Lanota, Dynamix and KALPA. Moreover I've been featured in AD:PIANO VIII by collaborating with another FA, Raimukun.
+> 
+> If I'd have to recommend a song to represent my identity, I'd choose '1,2,3,4! Overclocked Edition'. (You can find this song in the FA list!) This song is a rather long at over 7 minutes, but it's a mix of many genres, and also it displays my maximum ability when I was composing it!
+
+[Cansol](https://osu.ppy.sh/beatmaps/artists/407), who composed [Train of Thought](https://osu.ppy.sh/beatmapsets/2155531#taiko/4543467) for the Round of 16 tiebreaker, had the following to say.
+
+[**You can find the Korean version of this interview here.**](https://gist.github.com/Walavouchey/e080dda5040b74a16b412fd6b2e3f3e5)
+
+---
+
+What was the train of thought behind *Train of Thought*? What kind of concept did you initially have for it, and did it evolve in any way during the production process?
+
+> Since I started composing not long after I had just been discharged from military service, I wanted to write a song that expressed wildly how I felt while serving in there! I thus pondered about what kind of concept would fit the feelings, and I found out about the expression 'Train of Thought'.
+> 
+> For that reason, I applied the two meanings to 'Train of Thought' and contained them into the song.
+> 
+> First, I focused on the 'flow of consciousness' part.
+> 
+> However, it's not just improvised; I pondered how could I write the song which would fit and be interesting, and tried to base it on simple ideas. In the process, I even drew inspiration from the saying 'lose one's train of thought'!
+> 
+> Second, I focused on the 'Train of Thought' in its literal sense.
+> 
+> This shows in latter part of the song, where I either created melodies inspired by the sounds trains make or adopted vibraphone melodies heard in South Korean subways.
+> 
+> Although I couldn't know the musical atmospheres of subway systems in all countries, I regarded the vibraphone is the proper instrument since I assume it's usually used for conveying comfortable information in real life.
+
+Were there any specific considerations regarding the song's style due to constraints of osu!taiko gameplay?
+
+> When I was composing, I felt like I might have been putting in a bit too many notes, so I asked the staff members "does this part sound a little too confusing?" midway through the process.
+> 
+> But they said that it was okay, awesome even, so I expressed what I wanted to do with confidence.
+> 
+> I thought the drum sounds were important for impact since the taiko and the drum are the same percussion instrument, and I wondered if all drum solos would be applied as notes in the beatmap. When I looked through it, the sounds were expressed perfectly.
```

Comment:
```suggestion
> One thing is that I thought the drum sounds were important for impact since the taiko and the drum are the same percussion instrument. I thought, there's no way they put notes on every single drum sound in the drum solos, right? But when I looked at the final beatmap, not a single drum sound was skipped.
```